controllers/basicAuth_test.go

```diff
@@ -129,7 +129,7 @@ func Test_BasicLoginHandler_LoginSuccess(t *testing.T) {
 	response := httptest.NewRecorder()
 	request := httptest.NewRequest(
 		"POST",
-		"/v1/auth/basic/login",
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.